### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by MMathisLab**
- update pyside6 
- only merge after napari-deeplabcut new version is released

### CODE REVIEW

**Comment by MMathisLab** [comment]
done? cc @n-poulsen

**Comment by n-poulsen** [comment]
@MMathisLab I believe we need a new `napari-deeplabcut` release (version `0.2.1.5`) including the updates made in [#115](https://github.com/DeepLabCut/napari-deeplabcut/pull/115) and [#118](https://github.com/DeepLabCut/napari-deeplabcut/pull/118) before we can merge this. 

There are a few bugs with `napari-deeplabcut==0.2.1.3` and `pyside6==6.4.2` that have been fixed.

**Comment by MMathisLab** [comment]
okay can you make a release?

File: setup.py
Comment (n-poulsen) [inline setup.py]: Update for the [latest release](https://pypi.org/project/napari-deeplabcut/#history)

```suggestion
            "napari-deeplabcut>=0.2.1.5",
```

**Review by n-poulsen — CHANGES_REQUESTED** [review]
Tested with Python 3.10 and 3.11. I found one issue; when selecting the "Show trails" checkbox, the napari window goes blank. This is easily solved by resizing the window, but I'll open an issue about this so people have somewhere to go for a quick fix (once this PR is merged).

Otherwise all good for me once the `napari-deeplabcut` version is updated!

---

I'm also getting a warning when installing the `apple_mchips` extra:

```
WARNING: deeplabcut 2.3.9 does not provide the extra 'apple-mchips'
```

I believe this is the same `pip` issue as we saw with `napari-deeplabcut` (changes to the way underscores are transformed into dashes), and we can solve it by renaming the extra to `apple-mchips`. Currently not an issue as tensorflow is still installed correctly, despite the warning.

File: setup.py
Comment (jeylau) [inline setup.py]:
```suggestion
            "napari-deeplabcut>=0.2.1.6",
```

**Comment by jeylau** [comment]
I've just pushed a new release (0.2.1.6) of napari-deeplabcut, that includes our latest fixes/improvements.
This is ready to merge after the setup.py 😊

**Comment by MMathisLab** [comment]
I guess now we can merge, or no?

**Comment by jeylau** [comment]
@MMathisLab, yes, that looks good to me!

**Review by n-poulsen — APPROVED** [review]

**Pull request merged**
